### PR TITLE
Factorize

### PR DESCRIPTION
**Pull request by sebavan**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14583/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14583/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14583/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14583/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14583/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14583/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14583/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14583/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14583/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

File: packages/dev/core/src/Maths/math.plane.ts [hunk 1]
```diff
@@ -189,8 +189,8 @@ export class Plane {
      * Note : the vector "normal" is updated because normalized.
```
Comment (Popov72) [inline packages/dev/core/src/Maths/math.plane.ts:189]: You should remove this comment, as it's not relevant anymore.

**Review by Popov72 — APPROVED** [review]

**Pull request merged**